app/controllers/ChoiceController.scala

```diff
@@ -111,7 +111,7 @@ class ChoiceController @Inject()(
           createdDateTime = Instant.now,
           updatedDateTime = Instant.now,
           sourceId = None,
-          choice.value
+          choice.toDeclarationType.get
```

Comment:
Using just `get` will causes the exceptions for `None`.

It's for all other places too except tests.

Reply:
Yep, but it cant happen (see logic above)

Reply:
Yeah, I found it. What do you think about extracting this value before to pass it to the method without option?

Reply:
I mean, instead of passing `Choice` to this method pass `DeclarationType`